tests/test_parser.py

```diff
@@ -297,6 +297,7 @@ def test_get_parent_env_from_scratch(self, tmpdir):
 
         assert not df1.envs
 
+    # pylint: disable=anomalous-backslash-in-string
     @pytest.mark.parametrize(('instr_value', 'expected'), [
         ('"name1"=\'value 1\' "name2"=myself name3=""',
          {'name1': 'value 1',
```

Comment:
I think this would disable the warning for the whole file from this line onwards.
It *might* work if you put it here instead:
```python
@pytest.mark.parametrize(('instr_value', 'expected'), [
    # pylint: disable=anomalous-backslash-in-string
    ('"name1"=\'value 1\' "name2"=myself name3=""',
```